### ISSUE TITLE
Personality for dragons

### ISSUE THREAD

**Issue by MiyuAmayu** [AlexModGuy/Ice-and-Fire-Suggestions#1305]
Hi i got another idea, what if dragons had more personality in them?
Example: 
- Being picky and eating f.e. only pork or chicken
- They will hate certain mobs (maybe the first one they see or get hit by or we get hit by them on their sight at stage 1) 
- playing with their siblings, --> if they dont have maybe with their "mother" (player obv.)
- fire dragons for ex. Will often find it as pleasure to sit on/in fire (maybe they will have hearts [like when u give ur dog cooked pork] or heal 
??SOMEHOW?? faster ) 
-^^^ maybe they will spit fire a little that would probably cause disaster
-when u wake up just like cats they could give u smth (if u keep them at home or smth) 
Or they could by any change give it to you if u pass them by
- Shedding (not personality)
- giving Ur dragon gifts like gold or silver

**Comment by TrueBlueFlare** [comment]
Ooh I actually quite like this idea. Maybe even different dragons could have randomly assigned personality traits?

**Comment by CTH999** [comment]
I agree with TrueBlueFlare. There should be a collection of possible traits (and sub-traits) (For instance, "picky eater" (trait), leads to 2-5 "sub-traits", which are what foods they like!). Then, dragons can get a random selection of traits, and if you hatch multiple at once, it is likley they will influence one another, leading to certain traits. So, if you want specific traits, find dragons with ones you like, and then influence newborns to most likley have them, eventually only having traits _you_ want!

also, feeding them lots of different food when younger can _prevent_ picky eater, over-feeding a specific food can make them hate that one, feeding too many extra at once when older can lead to issues! (Stage 3+, feeding too much variety without slowly increasing variety can develop that trait)

**Comment by Evan776** [comment]
Ooh yes! Literally why I was coming here haha. I even think small things like some dragons are more likely to want to fly when wandering, or some are lazy and will want to sleep during the day. This could possibly happen with age too. I think it would make their behavior more interesting.
I also love the idea of them shedding!

**Comment by WamoZero** [comment]
Nice, maybe they could also go on a rampage if the owner dies in their sight?
